account_invoice_validation_queued/models/account_invoice.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+try:  # the try/except statements can be removed on v12
```

Comment:
```suggestion
try:  # TODO the try/except statements can be removed on v12
```

Reply:
This is not a TODO, as for v11 this is totally done. Even in v12, this is something harmless, so I'm not going to mark it that way.